src/Components/Components/src/ComponentBase.cs

```diff
@@ -129,7 +130,19 @@ protected virtual bool ShouldRender()
         /// <summary>
         /// Method invoked after each time the component has been rendered.
         /// </summary>
-        protected virtual void OnAfterRender()
+        /// <param name="firstTime">
+        /// Set to <c>true</c> if this is the first time <see cref="OnAfterRender(bool)"/> has been invoked
+        /// on this component instance; otherwise <c>false</c>.
+        /// </param>
+        /// <remarks>
+        /// <para>
+        /// The <see cref="OnAfterRender(bool)"/> and <see cref="OnAfterRenderAsync(bool)"/> lifecycle methods
+        /// are useful for performing interop, or interacting with values recieved from <c>@ref</c>.
+        /// Use the <paramref name="firstTime"/> parameter to ensure that initialization work is only performed
+        /// once.
+        /// </para>
```

Comment:
Shouldn't need to wrap a single block of text in `<para>`&hellip;`</para>` unless you're expecting more in the `<remarks/>`. (Here and later in the PR.)